server/bin/import_test_data.rb

```diff
@@ -189,12 +189,14 @@ def create_product(cp, owner, product)
   attrs['variant'] = variant
   attrs['arch'] = arch
   attrs['type'] = type
-  product_ret = cp.create_product(owner['name'], id, name, {:multiplier => multiplier,
-                                             :attributes => attrs,
-                                             :dependentProductIds => dependent_products,
-                                             :relies_on => relies_on})
-  puts "product name: " + name + " version: " + version + \
-       " arch: " + arch + " type: " + type
+  product_ret = cp.create_product(owner['name'], id, name, {
+    :multiplier => multiplier,
+    :attributes => attrs,
+    :dependentProductIds => dependent_products,
+    :relies_on => relies_on
+  })
+
+  print "product name: " + name + " version: " + version + " arch: " + arch + " type: " + type + "\n"
```

Comment:
Use interpolation here.  It will read better.

`print "product name: #{name} version: #{version} arch: #{arch} type: #{type}\n"`